cilium/tetragon
# feat: develop Tetragon Enterprise page

**Describe what changes this pull request brings**

- Add Enterprise page
- Update `Welcome to the community!` section

**Steps to check:**
1. Open [preview](https://deploy-preview-3103--tetragon.netlify.app/enterprise/)
2. Make sure that everything looks and works as expected
